growth/mailerAPI.js

```diff
@@ -0,0 +1,55 @@
+//API code from Growth Team: This code is to be implemented where a diiferent team would like to use
+//our API to send out emails
+// file name: mailerAPI.js
+function sendMail(emailAddress, emailSubject, emailHtmlPATH) {
+    const data = {
```

Comment:
do you run any validation check on the inputs? such as to, subject, `isschedule` ? 
what if I'll input wrong email? or send `isSchedule: true` without `timeToSend`?